test/e2e/networking.go

```diff
@@ -29,6 +29,60 @@ import (
 	. "github.com/onsi/gomega"
 )
 
+var _ = BeforeSuite(func() {
+	//Assert basic external connectivity.
+	//Since this is not really a test of kubernetes in any way, we
+	//leave it as a pre-test assertion, rather than a Ginko test.
+	resp, err := http.Get("http://google.com")
+	if err != nil {
+		Failf("Unable to connect/talk to the internet: %v", err)
+	}
+	if resp.StatusCode != http.StatusOK {
+		Failf("Unexpected error code, expected 200, got, %v (%v)", resp.StatusCode, resp)
+	}
+})
+
+func LaunchNetTestPodPerNode(nodes *api.NodeList, name string, c *client.Client, ns string) []string {
```

Comment:
Maybe move this function to the test/e2e/util.go? It's useful function which starts pod on each node. As a parameter use docker image name.